lib/vmdb/loggers/multicast_logger.rb

```diff
@@ -0,0 +1,31 @@
+class MulticastLogger < Logger
+  attr_accessor :loggers
+
+  def initialize(*loggers)
+    require 'set'
+    @loggers = Set.new(loggers)
+    @level   = DEBUG
+  end
+
+  def level=(new_level)
+    loggers.each { |l| l.level = new_level }
+    super
+  end
+
+  def filename
+    loggers.first.filename
+  end
+
+  [:log_backtrace, :log_hashes].each do |method|
+    define_method(method) do |*args|
+      loggers.map { |l| l.send(method, *args) }.first
```

Comment:
I don't think we care right now.  We only have one place using it and it is always backed by one of our loggers.